src/Storage/Adapter/RedisCluster.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+/**
+ * This file is part of the Phalcon Framework.
+ *
+ * (c) Phalcon Team <team@phalcon.io>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Phalcon\Storage\Adapter;
+
+use DateInterval;
+use Exception as BaseException;
+use Phalcon\Storage\Exception as StorageException;
+use Phalcon\Storage\SerializerFactory;
+use Phalcon\Support\Exception as SupportException;
+use RedisCluster as RedisService;
+
+use Throwable;
+
+use function defined;
+use function is_bool;
+use function is_int;
+use function mb_strtolower;
+
+/**
+ * Redis adapter
+ *
+ * @property array $options
+ */
+class RedisCluster extends AbstractAdapter
+{
+    /**
+     * @var string
+     */
+    protected string $prefix = 'ph-reds-';
```

Comment:
This is the same key as the Redis adapter. Can we change this please? Perhaps `ph-redc-` (for cluster)

Reply:
changed